bin/sl-deploy.js

```diff
@@ -56,16 +59,19 @@ while ((option = parser.getopt()) !== undefined) {
       local = true;
       break;
     default:
-      console.error('Invalid usage (near option \'%s\'), try `%s --help`.',
-        option.optopt,
-        $0);
+      console.error(GLB.t('msgInvalidUsage1', {
+        phOption: option.optopt,
```

Comment:
ph and msg are introduced for the following two (minor) reasons.  i put relatively less emphasis on readability.  will consider it.
1. ph -- give human translator a clear indication
2. msg, ph -- indicator in source code